jobs/build/ocp/Jenkinsfile

```diff
@@ -710,7 +722,7 @@ ${failed_map}
                     return false  // cause waitUntil to loop again
                 } else if ( resp == "CONTINUE" ) {
                     echo "User chose to build fails are OK."
```

Comment:
wording

Reply:
Wow... I'm gonna blame autocorrect there. Sure. That's it...
Updated